src/Integration/LocalServices/RuleSetInspector.cs

```diff
@@ -48,15 +48,65 @@ public RuleSetInspector(IServiceProvider serviceProvider, params string[] userRu
             ruleSetSearchDirectories.Add(GetStaticAnalysisToolsDirectory());
         }
 
+        /// <summary>
+        /// <see cref="IRuleSetInspector.FindConflictingRules(string, string)"/>
+        /// </summary>
+        public RuleSet FixConflictingRules(string baselineRuleSetPath, string targetRuleSetPath)
```

Comment:
Inconsistent casing of "baseline" in other occurrences in this file - some are still "baseLine".